### PR TITLE
App migrations

### PR DESCRIPTION
**Pull request by perryr16**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,17 +10,27 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-ActiveRecord::Schema.define(version: 20200512205827) do
-
-# ActiveRecord::Schema.define(version: 20200512020737) do
-# This was a merge conflict but wasnt sure if deleting it was the best option. Otherwise version above is from mine
-
+ActiveRecord::Schema.define(version: 20200513215451) do
```
Comment (perryr16) [inline db/schema.rb:14]: seems like everytime we make a migration were going to have to deal with one of these conflicts

File: db/schema.rb [hunk 2]
```diff
@@ -10,17 +10,27 @@
...
-  create_table "favorites", force: :cascade do |t|
+  create_table "apps", force: :cascade do |t|
+    t.string "name"
+    t.string "address"
+    t.string "city"
+    t.string "state"
+    t.string "zip"
+    t.string "phone_number"
+    t.string "description"
+    t.boolean "approved"
```
Comment (perryr16) [inline db/schema.rb:26]: if you create an app, and set `approved = true` we should be in business. You'll also have to use the join table PetApps to make sure each app is associated with a pet and vise versa (see the playlistsongs from the lesson)

File: db/schema.rb (hunk 1 above)
Comment (muydanny) [reply db/schema.rb:14]: Yeah Im assuming we use the most recent one.

File: db/schema.rb (hunk 2 above)
Comment (muydanny) [reply db/schema.rb:26]: Coool coool coool

**Pull request merged**